src/main/resources/css/layout.css

```diff
@@ -0,0 +1,1073 @@
+	text-decoration:none;
+	background: #777777;
+	color: #fff;
+	-moz-border-radius: 3px;
+	-webkit-border-radius:3px;
+	border-radius:3px;
+}
+	
+#nav ul ul {
+	margin:20px 0 0 0;
+	width:180px;
+	text-transform:none;
+}
+	
+#nav ul ul ul {
+	margin:-1px 0 0 40px;
+	text-transform:none;
+	border-bottom:none;
+}
+	
+#nav ul ul li {
+	margin-left:0px;
+}
+	
+#nav ul ul li a {
+	margin-left:0px;
+	background:none;
+	width:160px;
+	padding:10px 10px 8px 10px;
+}
+	
+#nav ul ul li a:hover {
+	background:#e2e2e2;
+}
+	
+#nav ul ul {
+	background:#161519;
+	border-left:1px solid #909090;
+	border-right:1px solid #909090;
+}
+
+#nav ul ul ul {
+	border:1px solid #909090;
+}
+	
+#nav ul ul li {
+	border-bottom:1px solid #909090;
+}
+
+
+/************************************************************************
+-) MISC
+*************************************************************************/
+
+
+/*** CHECKLIST ***/
+
+ul.checklist { 
+	list-style:none !important; 
+	margin-left:0px; 
+	}
+
+ul.checklist li { 
+	background: url(check.png) 0 1px no-repeat; 
+	padding-left:30px;
+	line-height: 22px; 
+}
+
+
+.left {
+	float: left;
+}
+
+.right {
+	float: right;
+}
+
+.bottom {
+	margin-top: 40px;
+}
+
+.category {
+	padding: 6px 7px 7px 35px;
+}
+
+.category a {
+	font-size: 11px;
+	font-weight: bold;
+	color: #555555;
+	text-decoration: none
+}
+
+.hire_me {
+	font-family: Arial;
+	font-size: 13px;
+	font-weight: bold;
+	color: #555555;
+	margin-top: 20px;
+}
+
+.rounded {
+	-moz-border-radius: 3px;
+	-webkit-border-radius:3px;
+	border-radius:3px;
+}
+
+.big_rounded {
+	-moz-border-radius: 5px;
+	-webkit-border-radius:5px;
+	border-radius:5px;
+}
+
+body .alignleft {
+	float:left;
+	margin:10px 20px 0 0;
+	display: block;
+}
+
+.alignleft img, .alignright img{
+	display:block;
+} 
+
+.hr { 
+	clear: both;
+	padding: 0;
+	margin: 0 0 1em;
+	height: 5px;
+	background: transparent url("hr-bg.gif") repeat-x 0 0; 
+}
+
+.padding_item_first {
+	padding-top: 30px;
+	margin-bottom: 80px;
+}
+
+.padding_item {
+	padding-top: 150px;
+	margin-bottom: 80px;
+}
+
+.padding_service {
+	padding-top: 95px;
+	margin-bottom: 80px;
+}
+
+.margin_top {
+	margin-top: 40px;
+}
+
+.margin_bottom {
+	margin-bottom: 40px;
+}
+
+.margin_break {
+	margin-top: 110px;
+}
+
+.margin_footer_break {
+	margin: 20px 0 20px 0;
+}
+
+
+
+
+/*
+	960 Grid System ~ Core CSS.
+	Learn more ~ http://960.gs/
+
+	Licensed under GPL and MIT.
+*/
+
+/* `Containers
+----------------------------------------------------------------------------------------------------*/
+
+.container_12,
+.container_16 {
+	margin-left: auto;
+	margin-right: auto;
+	width: 960px;
+}
+
+/* `Grid >> Global
+----------------------------------------------------------------------------------------------------*/
+
+.grid_1,
+.grid_2,
+.grid_3,
+.grid_4,
+.grid_5,
+.grid_6,
+.grid_7,
+.grid_8,
+.grid_9,
+.grid_10,
+.grid_11,
+.grid_12,
+.grid_13,
+.grid_14,
+.grid_15,
+.grid_16 {
+	display: inline;
+	float: left;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [display:inline has no effect on floated elements (but may be used to fix the IE6 double-margin bug).](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2139367645/issues/source?bid=3059308&fileBranchId=3150237#l331)